app/_how-tos/allow-multiple-authentication.md

```diff
@@ -45,8 +73,139 @@ cleanup:
+```sh
+export DECK_ANONYMOUS_CONSUMER=434772ef-af0c-4227-a33b-76e33b9fd7df
+```
+
+## 2. Create Consumers
+
+Create three Consumers, including an `anonymous` Consumer.
+The `anonymous` Consumer doesn't correspond to any real user, and will only serve as a fallback:
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: anonymous
+      id: ${anonymous_consumer}
+    - username: Dana
+    - username: Mahan
+variables:
+  anonymous_consumer:
+    value: $ANONYMOUS_CONSUMER
+{% endentity_examples %}
+
+We're going to assign a different authentication type to each Consumer later.
+
+## 3. Set up authentication
+
+Add the Key Auth and Basic Auth plugins to the `example-service` Gateway Service, and set the `anonymous` fallback to the Consumer we created earlier:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: key-auth
+      service: example-service
+      config:
+        hide_credentials: true
+        anonymous: ${anonymous_consumer}
+    - name: basic-auth
+      service: example-service
+      config:
+        hide_credentials: true
+        anonymous: ${anonymous_consumer}
+variables:
+  anonymous_consumer:
+    value: $ANONYMOUS_CONSUMER
+{% endentity_examples %}
+
+## 4. Test with anonymous Consumer
+
+You now have authentication enabled on the Gateway Service, but the `anonymous` Consumer also allows requests from unauthenticated clients.
+
+Check without credentials:
+
+{% validation request-check %}
+url: '/anything'
+status_code: 200
+{% endvalidation %}
+
+Check with nonsense credentials:
+
+{% validation request-check %}
+url: '/anything?apikey=nonsense'
+status_code: 200
+{% endvalidation %}
+
+In both cases, you should get a 200 response, as the `anonymous` Consumer is allowed.
+
+## 5. Configure credentials
+
+Configure different credentials for the two named users: basic auth for `Dana`, and key auth for `Mahan`:
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: Dana
+      basicauth_credentials:
+        - username: Dana
+          password: dana
+    - username: Mahan
+      keyauth_credentials:
+        - key: mahan
+{% endentity_examples %}
+
+
+## 6. Add Request Termination to the anonymous Consumer
+
+The anonymous Consumer gets no credentials, as we don't want unauthenticated users accessing our Gateway Service.
+Instead, you can configure the Request Termination plugin to handle anonymous Consumers and redirect their requests with a `401`:
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: anonymous
+      plugins:
+        - name: request-termination
+          config:
+            status_code: 401
+            message: '"Error - Authentication required"'
+{% endentity_examples %}
+
+## 7. Validate authentication
+
+Let's check that authentication works.
+
+Try to access the Gateway Service via the `/anything` Route using a nonsense API key:
+
+{% validation request-check %}
+url: '/anything?apikey=nonsense'
+status_code: 401
+{% endvalidation %}
```

Comment:
This gave me the same error as the other "nonsense" validation check, but the rest of the validation worked.